scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -244,6 +243,47 @@ function exec_to_bash_or_python_command_if_specified() {
     fi
 }
 
+function check_uid_gid() {
+    if [[ $(id -g) == "0" ]]; then
+        return
+    fi
+    if [[ $(id -u) == "50000" ]]; then
+        >&2 echo
```

Comment:
I turned it into warning only if 'airflow' user is used and GID is not 0. This should be quite nice for the users:

* existing helm charts will work
* it's rather OK to run airflow with any GID as long as you are never going to use the arbitrary UID for already created directories
* we clearly warn the users and they will see it in their log files that GID=0 is the "recommended" way.